Jenkinsfile

```diff
@@ -134,7 +134,7 @@ pipeline {
 
       agent {
         docker {
-          image "us.gcr.io/logdna-k8s/node:12-ci"
+          image "us.gcr.io/logdna-k8s/node:17-ci"
```

Comment:
I would use 16-ci here since 17 is not LTS. It might still get breaking changes

Reply:
ack

Reply:
To that point, I'll set `.nvmrc` to 16 as well.  Thanks!